### ISSUE TITLE
Error Fooocus EMERGENCY

### ISSUE THREAD

**Issue by osquitarxp** [MoonRide303/Fooocus-MRE#99]
Please help, I was working with this great version that seemed fantastic and intuitive to use. Now I get an error when starting:

missing {'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids'}
Refiner model loaded: C:\Difusion\Focus Mre\Fooocus-MRE\models\checkpoints\sd_xl_refiner_1.0_0.9vae.safetensors
Exception in thread Thread-2 (worker):
Traceback (most recent call last):
File "C:\Difusion\Focus Mre\Fooocus-MRE-env\lib\threading.py", line 1016, in _bootstrap_inner
self.run()
File "C:\Difusion\Focus Mre\Fooocus-MRE-env\lib\threading.py", line 953, in run
self._target(*self._args, **self._kwargs)
File "C:\Difusion\Focus Mre\Fooocus-MRE\modules\async_worker.py", line 40, in worker
import modules.default_pipeline as pipeline
File "C:\Difusion\Focus Mre\Fooocus-MRE\modules\default_pipeline.py", line 269, in
refresh_everything(
File "C:\Difusion\Focus Mre\Fooocus-MRE\modules\default_pipeline.py", line 258, in refresh_everything
virtual_memory.try_move_to_virtual_memory(xl_refiner.unet.model)
File "C:\Difusion\Focus Mre\Fooocus-MRE-env\lib\site-packages\torch\utils_contextlib.py", line 115, in decorate_context
return func(*args, **kwargs)
File "C:\Difusion\Focus Mre\Fooocus-MRE\modules\virtual_memory.py", line 185, in try_move_to_virtual_memory
move_to_virtual_memory(model, comfy_unload)
File "C:\Difusion\Focus Mre\Fooocus-MRE-env\lib\site-packages\torch\utils_contextlib.py", line 115, in decorate_context
return func(*args, **kwargs)
File "C:\Difusion\Focus Mre\Fooocus-MRE\modules\virtual_memory.py", line 75, in move_to_virtual_memory
model_management.unload_model()
AttributeError: module 'comfy.model_management' has no attribute 'unload_model'

I must say that I also have the original version of fooocus installed and it works ok without errors, I also have the stable diffusion 1111 and it is ok, but this version of foocus MRE worked fine for a few days, and then it failed... I deleted everything and returned it to install from scratch step by step.... and if sending the same error.... ! someone help me please!

**Comment by osquitarxp** [comment]
@MoonRide303 help please

**Comment by MoonRide303** [comment]
@osquitarxp v2.0.19 works on my test machines. Could you describe step by step what did you do, so I could reproduce the problem?

You can also try replacing content of `Fooocus-MRE` folder with files from [any earlier tag](https://github.com/MoonRide303/Fooocus-MRE/tags), and then start without updating via `run-mre-no-update.bat` - I could look into differences between specific versions (on which it stopped working for you), then.

**Comment by MoonRide303** [comment]
@osquitarxp Okay, I think I've found it (the stack trace you've provided enough information), and pushed the fix (commit [b028acf](https://github.com/MoonRide303/Fooocus-MRE/commit/b028acfe1e78a9632a835da615735afd35e2a4a2)) - could you please try the latest version, and see if it helped?

**Comment by MoonRide303** [comment]
I applied workaround in v2.0.19 (commit [d9a09b4](https://github.com/MoonRide303/Fooocus-MRE/commit/d9a09b45b599aa47d09fa470556c6f63d05142cc), disables Fooocus Virtual Memory) until I figure out why Fooocus Virtual Memory stopped working in MRE v2.0.18.

**Comment by osquitarxp** [comment]
@MoonRide303 
Thank you for writing, it is very valuable for us to be able to receive your attention and help, I was about to take action as you gave me instructions, and I saw that the version was updated to v2.0.19, now without having made any modifications (I It seems that it updated itself), the problem was fixed and now it is running without problems, in any case, again thank you for your help and I am fascinated with this version of fooocus MRE, I thank you very much. blessings at home.

Gracias, por escribir, es muy valioso para nosotros poder recibir su atencion y ayuda, estaba a punto de tomar medidas segun me diste las indicaciones, y vi que se actualizo la version a v2.0.19, ahora sin que haya hecho ninguna modificacion (me parece que se auto actualizo), se arreglo el problema y ahora esta corriendo sin problemas, en todo caso, nuevamente agradecer tu ayuda y estoy fascinado con esta version de fooocus MRE, te agradesco mucho. bendiciones en casa.
